### PR TITLE
Mani/minify sass

### PR DESCRIPTION
**Pull request by maniart**
- Minify css, reduce file size by `40kb`

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -35,16 +35,18 @@ gulp.task('createAdkTemplatesList', function() {
 gulp.task('sass', function() {
   return gulp.src('scss/adk.scss')
     .pipe($.sass({
-      includePaths: sassPaths,
-      outputStyle: 'compressed'
```
Comment (maniart) [inline gulpfile.js]: Through experimentation I found that `cssWrap` undoes the `outputStyle: 'compressed'` option, effectively un-minifying the output css.

File: gulpfile.js [hunk 2]
```diff
@@ -35,16 +35,18 @@ gulp.task('createAdkTemplatesList', function() {
...
+      selector: '.adk',
+    }))
     .pipe($.autoprefixer({
       browsers: ['last 2 versions', 'ie >= 9']
     }))
-    .pipe(cssWrap({
-      selector: '.adk', // Wrap all Design Kit CSS in .adk class so we can scope it
-    }))
+    // Minify the output
+    .pipe($.cleanCss())
```
Comment (maniart) [inline gulpfile.js]: Adding this gulp plugin as the last step ensures that output is minifies after:
- compiling to css
- autoprefixing

Comment (expandrew) [reply gulpfile.js]: Ahh nice, so any minification that the `gulp-sass` step was doing was being overridden by the `cssWrap` step? Bummer ... ugh

But this seems like it's a good way to explicitly minify after everything is done, works for me 👍

Comment (maniart) [reply gulpfile.js]: @expandrew Super weird. You'd expect gulp to pass output stream of one process to the next. Not sure why it doesn't happen.

**Pull request merged**